.buildkite/pipeline.yml

```diff
@@ -1,3 +1,57 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+env:
+  SETUP_MAGE_VERSION: '1.14.0'
+  SETUP_GOLANG_VERSION: '1.20.4'
+
 steps:
-  - label: "Example test"
-    command: echo "Hello!"
+  - label: ":golangci-lint: Checks formatting / linting"
+    command:
+      - ".buildkite/scripts/pre-install-command.sh 'mage -debug check'"
```

Comment:
I would follow the same approach as @sharbuz, create separate scripts for this. So you don't need to do `eval` in the pre-install-command.sh
https://github.com/elastic/fleet-server/pull/2543